bin/reth/src/test_eth_chain/runner.rs

```diff
@@ -205,9 +205,10 @@ pub async fn run_test(path: PathBuf) -> eyre::Result<()> {
                 tracing::trace!("Our storage:{:?}", storage);
                 for (address, test_account) in state.iter() {
                     // check account
-                    let our_account = tx.get::<tables::PlainAccountState>(*address)?.ok_or(
-                        eyre!("Account is missing: {address} expected: {:?}", test_account),
-                    )?;
+                    let our_account =
+                        tx.get::<tables::PlainAccountState>(*address)?.ok_or_else(|| {
+                            eyre!("Account is missing: {address} expected: {:?}", test_account)
+                        })?;
```

Comment:
We should use `wrap_err` here: https://docs.rs/eyre/latest/eyre/trait.WrapErr.html#tymethod.wrap_err

Reply:
@onbjerg Thanks for your comment! I'm really happy to get your feedback.

Even though I'm not familiar with `eyre`, after reviewing what is `wrap_err`, I think we couldn't use `wrap_err` here. 

```
tx.get::<tables::PlainAccountState>(*address)?
```
w/ `?`, it will return an error if the Result is error so it's now an Option.
ref:
https://github.com/paradigmxyz/reth/blob/e6ca4c56c6a314e5553656e5b62be3bca6801b85/crates/storage/db/src/implementation/mdbx/tx.rs#L74-L83

```
ok_or_else(|| { eyre!("Account is missing: {address} expected: {:?}", test_account) })?;
```
will handle an case when the option is `None`.

If I misunderstood something, please let me know. Thanks!

Reply:
you're right, we have an `Option::ok_or_else`, so using `eyre!` is fine here.